README.md

```diff
@@ -42,6 +42,12 @@ To use from Go, use `go get` to grab the latest version of the library
 
     $ go get github.com/pulumi/pulumi-aws/sdk/go/...
 
+### .NET
+
+To use from .NET, install using `nuget`:
```

Comment:
I realize it's a NuGet package, but should we just say the same command as we're showing below, like we do for `go get`?

```suggestion
To use from .NET, install using `dotnet add package`:
```

Reply:
I followed the same example as TypeScript and Python here by mentioning the package manager - Go has no package manager - thus why it says Go get I reckon?

Reply:
When it's styled as code (with backticks) it makes me think it's a command like `npm`, `pip`, `yarn`, `go get`. If we were just saying the package manager, wouldn't we refer to these as NPM and PyPI?

Reply:
Got it! Will update :)